crates/chia-sdk-bindings/src/puzzles.rs

```diff
@@ -292,3 +293,93 @@ pub fn standard_puzzle_hash(synthetic_key: PublicKey) -> Result<Bytes32> {
 pub fn cat_puzzle_hash(asset_id: Bytes32, inner_puzzle_hash: Bytes32) -> Result<Bytes32> {
     Ok(CatArgs::curry_tree_hash(asset_id, inner_puzzle_hash.into()).into())
 }
+
+#[derive(Clone)]
+pub struct StreamedCat {
+    pub coin: Coin,
+    pub asset_id: Bytes32,
+    pub proof: LineageProof,
+    pub inner_puzzle_hash: Bytes32,
```

Comment:
Is there a way to turn this into `StreamedCatInfo` and calculate the inner puzzle hash on the fly? Similar to `DidInfo`